oiva-core/src/test/java/fi/minedu/oiva/backend/core/it/BaseIT.java

```diff
@@ -67,6 +68,8 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @Import(EmbeddedRedisConfig.class)
+// We are not interested in log files when running integration tests
+@TestPropertySource(properties = {"logging.path = /tmp"})
```

Comment:
Tätä ei tarvita, kun application-it.yml tiedostossa testi logit ohjataan jo target/log/spring.log tiedostoon ja hyvä jos asiat pysyy projektin sisällä. Ja joskus voi olla tarvetta etsiä esim. jotain tuosta testiajon logista.

Reply:
Hyvä huomio. Lisään puuttuvan logging.path-propertyn it.yml-tiedostoihin.